capstone-app/src/app/app.component.scss

```diff
@@ -19,10 +19,60 @@
 }
 
 .title {
-  margin-top: 0;
-  margin-bottom: 30px;
   font-family: 'Open Sans', sans-serif;
-  font-size: 60px;
-  font-weight: 590;
+  font-size: 4vh;
```

Comment:
Tying font size to viewscreen height is very unusual, I'd recommend sticking to pixels. And font-weight below seems awful low. What is the effect of that?

Reply:
The reason why this is vh is because it's dealing with our logo (GMapr) and is responding to the toolbar changing size. Is this a must change?

Reply:
As for the font-weight, it actually didn't have effect on things so just removed it

Reply:
The preferred method of dealing with this is with media queries, but I can live with vh.

Reply:
Ok. In the meantime I'll look into switching it.